services/core-web/src/components/mine/Tailings/MineTailingsTable.js

```diff
@@ -35,98 +59,160 @@ export class MineTailingsTable extends Component {
-            {this.props.itrmExemptionStatusCodeHash[text] || Strings.EMPTY_FIELD}
-          </div>
+        key: "consequence_classification",
+        render: (record) => (
+          <Typography.Text>
+            {CONSEQUENCE_CLASSIFICATION_CODE_HASH[record.consequence_classification]}
+          </Typography.Text>
         ),
       },
       {
-        title: "Engineer of Record",
-        dataIndex: "engineer_of_record",
-        render: (text) => (
-          <div title="Engineer of Record">{text ? text.party.name : Strings.EMPTY_FIELD}</div>
-        ),
-      },
-      {
-        title: "Latitude",
-        dataIndex: "latitude",
-        render: (text) => <div title="Latitude">{text || Strings.EMPTY_FIELD}</div>,
-      },
-      {
-        title: "Longitude",
-        dataIndex: "longitude",
-        render: (text) => <div title="Longitude">{text || Strings.EMPTY_FIELD}</div>,
-      },
-      {
-        title: "Notes",
-        dataIndex: "notes",
-        render: (text) => <div title="Notes">{text || Strings.EMPTY_FIELD}</div>,
-      },
-      {
-        key: "operations",
-        render: (text, record) => {
+        title: "",
+        dataIndex: "edit",
+        render: () => {
           return (
-            <div align="right">
-              <AuthorizationWrapper permission={Permission.EDIT_TSF}>
-                <Button
-                  type="primary"
-                  size="small"
-                  ghost
-                  onClick={(event) =>
-                    this.props.openEditTailingsModal(event, this.props.handleEditTailings, record)
-                  }
-                >
-                  <img src={EDIT_OUTLINE_VIOLET} alt="Edit TSF" />
-                </Button>
+            <div title="" align="right">
+              <AuthorizationWrapper>
+                <Tooltip title="Edit Dam page does not yet exist">
+                  <Button type="primary" size="small" ghost onClick={() => {}}>
+                    <img src={EDIT_OUTLINE_VIOLET} alt="Edit Dam" />
+                  </Button>
+                </Tooltip>
               </AuthorizationWrapper>
             </div>
           );
         },
       },
     ];
-
     return (
       <CoreTable
-        condition={this.props.isLoaded}
-        dataSource={this.transformRowData(this.props.tailings)}
-        columns={columns}
+        columns={expandedColumns}
+        dataSource={expandedRecord.dams}
         tableProps={{
-          align: "center",
+          size: "small",
           pagination: false,
+          rowKey: (record) => record.dam_guid,
+          className: "tailings-nested-table",
         }}
+        condition={props.isLoaded}
       />
     );
-  }
-}
+  };
+
+  return (
+    <CoreTable
+      condition={props.isLoaded}
+      dataSource={transformRowData(props.tailings)}
+      columns={columns}
+      tableProps={{
+        align: "center",
+        pagination: false,
+        expandable: !IN_PROD() ? { expandedRowRender } : null,
```

Comment:
@taraepp Here's an example of the feature flag to keep this from showing in production.